analytical/tests/test_tag_google_analytics_gtag.py

```diff
@@ -0,0 +1,48 @@
+"""
+Tests for the Google Analytics template tags and filters, using the new gtag.js library.
+"""
+
+from django.http import HttpRequest
+from django.template import Context
+from django.test.utils import override_settings
+
+from analytical.templatetags.google_analytics_gtag import GoogleAnalyticsGTagNode
+from analytical.tests.utils import TagTestCase
+from analytical.utils import AnalyticalException
+
+
+@override_settings(GOOGLE_ANALYTICS_GTAG_PROPERTY_ID='UA-123456-7')
+class GoogleAnalyticsTagTestCase(TagTestCase):
+    """
+    Tests for the ``google_analytics_js`` template tag.
+    """
+
+    def test_tag(self):
+        r = self.render_tag('google_analytics_gtag', 'google_analytics_gtag')
+        self.assertTrue("""<script async src="https://www.googletagmanager.com/gtag/js?id=UA-123456-7"></script>""" in r, r)
```

Comment:
This line (and [line 38](#diff-5faab0fb53bde9f058fe071a8cbeccdfR28)) is what flake8 complains about. The shortest form that I still find readable I came up with is:
```
        self.assertTrue('<script async src="https://www.googletagmanager.com/gtag/js?id=UA-123456-7">'
        	'</script>' in r, r)
```

Still 102 characters. Maybe add `# noqa: E501` here instead. @mbourqui @bittner what do you think?

Reply:
Splitting it on three lines seems to do the trick, without breaking the html code line.